profile/label_nodes.go

```diff
@@ -0,0 +1,97 @@
+package profile
```

Comment:
RE label_nodes.go - it would be nice to avoid the underscore if possible, via a shorter name. Vast majority of other files in pprof codebase don't have an underscore (other than _test.go files).

Reply:
I'm struggling to think of a shorter name... is `labelnodes.go` OK? `tagnodes.go`? `tagroot.go`? I was going off examples like `driver_focus.go`.

Reply:
Gone with `labelnodes.go` for now, but happy to change to whatever.

Reply:
I'm not a fan of driver_focus.go either - I think I'd either move that code to driver.go or have it as focus.go. The package name is already "driver" there so "driver_" as a prefix stutters IMO. Anyway, not suggesting to change THAT :)

For this PR, wdyt about internal/driver/tagroot.go? It's close to the flag name so I think it makes it easier to relate the code and the functionality. It doesn't cover the leaf case of course but oh well.